calico-cloud_versioned_docs/version-18/variables.js

```diff
@@ -1,22 +1,22 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.17.1',
+  releaseTitle: 'v3.18.0-1.1',
   prodname: 'Calico Cloud',
   prodnamedash: 'calico-cloud',
   baseUrl: '/calico-cloud',
   filesUrl: 'https://docs.calicocloud.io',
-  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.17.1',
+  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.18.0-1.1',
   tutorialFilesURL: 'https://docs.tigera.io/files',
   prodnameWindows: 'Calico Enterprise for Windows',
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.17.1-4',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.17.1-4',
+  cloudversion: 'v3.18.0-1.1-0',
```

Comment:
This and the next line need to end in a 1. They should be `v3.18.0-1.1-1`

Reply:
Done, thanks